src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -419,6 +419,14 @@ protected function getEnv($name)
         if (false !== $env = getenv($name)) {
             return $this->envCache[$name] = $env;
         }
+        if (false !== $i = strpos($name, '@')) {
```

Comment:
What about a default implementation of `GetEnvInterface`? and be explicit here (in terms of priority);

```php
if () {
   $service->getEnv();
} else {
   $default->getEnv();
}
```